exchanges/bitfinex/bitfinex_types.go

```diff
@@ -19,6 +19,30 @@ var AcceptedWalletNames = []string{"trading", "exchange", "deposit", "margin",
 // AcceptableMethods defines a map of currency codes to methods
 var AcceptableMethods = make(map[string]string)
 
+// MarginV2FundingData stores margin funding data
+type MarginV2FundingData struct {
+	Symbol    string
+	RateAvg   float64
```

Comment:
We like to have full/not-abbreviated fields, please change to `RateAverage` etc etc